### DISCUSSION TITLE
GitPage displaying 404 Site not found

### DISCUSSION THREAD

**Discussion by vikrammohan7** [community/community#23308] [Other Feature Feedback, Questions, & Ideas]
<p>I have a repository vikrammohan7/vikrammohan7.github.io.</p>
<p>A simple GitPage repository with a simple html page. </p>
<p>When I tried to access <a href="https://vikrammohan7.github.io" rel="nofollow noopener">https://vikrammohan7.github.io</a> or <a href="https://vikrammohan7.github.io/vikrammohan7" rel="nofollow noopener">https://vikrammohan7.github.io/vikrammohan7</a> or <a href="https://vikrammohan7/vikrammohan7.github.io" class="inline-onebox-loading" rel="nofollow noopener">https://vikrammohan7/vikrammohan7.github.io</a> all shows 404  <strong>There isn’t a GitHub Pages site here.</strong></p>
<p>Am I doing something wrong or am I missing something?</p>
<p>How do I fix this to get the link <a href="https://vikrammohan7.github.io" rel="nofollow noopener">https://vikrammohan7.github.io</a> to display my index.html?</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/vikrammohan7">@vikrammohan7</a>,</p>
<p>Welcome to the Github Community Forum!</p>
<p>Now it seems working. The fact is that github pages needs some minutes to create the instance and run the process, you only need to wait. </p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>

**Comment by vikrammohan7** [comment]
<p>Thank you so much it works now.</p>
<p>Just curious, how long will it normally takes to reflect the changes?</p>

**Comment by rajkamal8** [comment]
<p>how long actually anyone needs to wait</p>
<blockquote>
<p><a class="mention" href="/wabri">@wabri</a> wrote:</p>
</blockquote>
<blockquote>
<p>Hi <a class="mention" href="/vikrammohan7">@vikrammohan7</a>,</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>Welcome to the Github Community Forum!</p>
</blockquote>
<blockquote>
<p>Now it seems working. The fact is that github pages needs some minutes to create the instance and run the process, you only need to wait. </p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>-Gabriele-</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>
</blockquote>

**Comment by Wabri** [comment]
<p>Depends on the load of the servers, approximately from 2 minutes to 10. The first time that you publish the page need more time to analize and publish the site.</p>

**Comment by yuvarajsai** [comment]
<p>Even I’m facing the same problem…how much time does it take? <a class="mention" href="/wabri">@wabri</a> <a class="mention" href="/vikrammohan7">@vikrammohan7</a> ?</p>

**Comment by vikrammohan7** [comment]
<p>I did not check the site for the whole night, I posted this question and slept, checked the commends in the morning and realised it worked, so not sure when it started working.</p>

**Comment by vikrammohan7** [comment]
<p>From my understanding if there is no other error or problem in displaying the site, it would take an hour or so for the first time, then it would be 2-10 mins like <a class="mention" href="/wabri">@wabri</a> commended above.</p>

**Comment by yuvarajsai** [comment]
<p>but for me it’s more than 3 hours and still not even a single change</p>
